rest/etc/default/zapata.rc

```diff
@@ -1,13 +1,12 @@
-
 RUNNING_SHELL=$(readlink /proc/$$/exe)
 if [[ $RUNNING_SHELL == /bin/zsh* ]]
 then
 	BASH_SOURCE=${(%):-%x}
 fi
+
 ZAPATA_ENV_CWD=${BASH_SOURCE%/*}
 
-rcs=($(find /etc/zapata/rc.d/ -name "*.rc"))
-for rc_file in $rcs
+for rc_file in `find /etc/zapata/rc.d/ -name "*.rc"`
```

Comment:
No need for this change, just remove the extra `(` and `)` in `rcs=($(find /etc/zapata/rc.d/ -name "*.rc"))`.